benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -185,31 +185,40 @@ void GraphicsBenchmarkApp::InitKnobs()
         pResolution->SetIndent(1);
     }
 
-    GetKnobManager().InitKnob(&pKnobAluCount, "alu-instruction-count", /* defaultValue = */ 100, /* minValue = */ 100, 400);
+    GetKnobManager().InitKnob(&pKnobAluCount, "alu-instruction-count", /* defaultValue = */ 0, /* minValue = */ 0, 500);
```

Comment:
we have shaders with 0 instruction?

Reply:
the name is confusing, maybe better to change the name to something that is more accurately reflects what it does?